src/Explorer/Explorer.tsx

```diff
@@ -372,8 +372,8 @@ export default class Explorer {
         if (connectionInfo.status !== HttpStatusCodes.OK) {
           throw new Error(`Received status code: ${connectionInfo?.status}`);
         }
-        if (!connectionInfo?.data?.notebookServerUrl) {
-          throw new Error(`NotebookServerUrl is invalid!`);
+        if (!connectionInfo?.data?.notebookServerUrl && !connectionInfo?.data?.phoenixServiceUrl) {
```

Comment:
lets have dependency only on phoenix server url and remove the notebookserverurl